zoo-css-challenge/style.css

```diff
@@ -29,13 +30,14 @@ section {
   background-color: #f8f0fb;
 }
 
-img {
-  border: 5px solid #8075ff;
+#logo {
+  border: none;
+}
+#logo {
```

Comment:
```suggestion
```
This is a duplicate `#logo` selector — there is already one above on line 33 so we can delete this one